sandwine/_main.py

```diff
@@ -152,12 +152,19 @@ def parse_command_line(args: list[str], with_wine: bool):
         help="enable use of host X11 (CAREFUL!) (default: X11 disabled)",
     )
 
-    networking = parser.add_argument_group("networking arguments")
+    gpu_nvidia = parser.add_argument_group("GPU arguments")
+    gpu_nvidia.add_argument(
+        "--gpu-nvidia",
```

Comment:
I wonder if `--nvidia` or `--nvidia-gpu` would be better. I see the idea of opening a namespace here, but adjusting this will likely help usability. I don't expect any non-GPU Nvidia things in here anytime soon or ever, so maybe implying GPU is good enough. What do you think?